.vale.ini

```diff
@@ -0,0 +1,7 @@
+StylesPath = .github/styles
```

Comment:
I'd just use `styles/`, like in the [vale-boilerplate](https://github.com/errata-ai/vale-boilerplate).

Reply:
p.s. I guess they also implicitly recommend [using the CI folders](https://vale.sh/docs/topics/config/#stylespath)...